android/src/main/java/com/mapbox/reactnativemapboxgl/ReactNativeMapboxGLView.java

```diff
@@ -682,7 +726,13 @@ public CameraPosition getCameraPosition() {
     }
 
     public LatLngBounds getBounds() {
-        if (_map == null) { return new LatLngBounds.Builder().build(); }
+        if (_map == null) {
+            LatLng center = _initialCamera.build().target;
+            return new LatLngBounds.Builder()
+                    .include(center)
+                    .include(center)
```

Comment:
Oh, it's because including center twice works, there is no problem with that. We need two points provided to builder, and it is not complaining if both are same. I can leave comment in code if it will make difference.